src/tool_help.c

```diff
@@ -87,7 +87,7 @@ static const struct category_descriptors categories[] = {
   {"imap", "IMAP protocol options", CURLHELP_IMAP},
   /* important is left out because it is the default help page */
   {"misc", "Options that don't fit into any other category", CURLHELP_MISC},
-  {"output", "The output of curl", CURLHELP_OUTPUT},
+  {"output", "The filesystem output of curl", CURLHELP_OUTPUT},
```

Comment:
How about just `filesystem output` ? Short and sweet.

Reply:
ACK, will change it tomorrow